cmd/notary/integration_test.go

```diff
@@ -1526,3 +1527,447 @@ func TestWitness(t *testing.T) {
 		require.Error(t, err)
 	}
 }
+
+func TestClientTUFInitWithAutoPublish(t *testing.T) {
+	// -- setup --
+	setUp(t)
+
+	tempDir := tempDirWithConfig(t, "{}")
+	defer os.RemoveAll(tempDir)
+
+	server := setupServer()
+	defer server.Close()
+
+	tempFile, err := ioutil.TempFile("", "targetfile")
+	require.NoError(t, err)
+	tempFile.Close()
+	defer os.Remove(tempFile.Name())
+
+	var (
+		output       = ""
+		gun          = "MistsOfPandaria"
+		gunNoPublish = "Legion"
+
+		// This might be changed via the implementation, please be careful.
+		emptyList = "\nNo targets present in this repository.\n\n"
+	)
+	// -- tests --
+
+	// init repo with auto publish being enabled but with a malformed URL.
+	_, err = runCommand(t, tempDir, "-s", "For the Horde!", "init", "-p", gun)
+	require.Error(t, err, "Trust server url has to be in the form of http(s)://URL:PORT.")
+	// init repo with auto publish being enabled but with an unaccessible URL.
+	_, err = runCommand(t, tempDir, "-s", "https://notary-server-on-the-moon:12306", "init", "-p", gun)
```

Comment:
this is an awesome test!  Can we add a similar malformed URL and inaccessible URL test for an originally offline command, like `add`?  For `init` it's less applicable because the command would already talk to notary server without publishing to setup the timestamp.